### ISSUE TITLE
keeps loading but the radar is never created

### ISSUE THREAD

**Issue by ardahuseyinoglu** [thoughtworks/build-your-own-radar#320]
I try to run the app locally with this following commands:
`npm install`
`npm run dev`

I can view the page successfully at `http://localhost:8080/`. However, radar is not created when i paste an any public google sheet URL (and I've also tried the one that you shared as an example: https://docs.google.com/spreadsheets/d/1sL6j3eYxsiXMtgELi52pn6XMzk_L4ZtfxYr1KjNYZpo/edit#gid=0)

It keeps loading but the radar is never created.

Error I get from the browser console is:

<img width="503" alt="image" src="https://github.com/thoughtworks/build-your-own-radar/assets/32541095/6e2a8f59-75d7-4619-ae8c-48e963ddd843">

I also created client-id and api-key from Google Developers Console and set them as both system and user variables on Windows Environment Variables.

**Comment by devansh-sharma-tw** [comment]
Hi @ardahuseyinoglu , given the missing `client_id` error message, most likely node is not able to pick up the env vars you have setup. To confirm, you can try debugging/logging the values of `CLIENT_ID` and `API_KEY` variables in [googleAuth.js](https://github.com/thoughtworks/build-your-own-radar/blob/bc609c1aea597fdc717113d963759163dfdd9a2c/src/util/googleAuth.js#L5).

If its not getting picked up (possibly due to how node interacts with Windows env variables ?), you could try passing the variables while starting the application itself, like this: `CLIENT_ID=<client_id> API_KEY=<api_key> npm run dev`. Or you could [export them in the terminal session](https://github.com/thoughtworks/build-your-own-radar#more-complex-usage) (if you are using a `bash`-like terminal).

Hope this helps! Thanks!

**Comment by ardahuseyinoglu** [comment]
I'm sorry for the late reply. Enabling Google Sheets API solved the issue.
